release-notes/10.0/preview/preview1/sdk.md

```diff
@@ -1,14 +1,16 @@
-# .NET SDK in .NET 10 Preview 1 - Release Notes
-
-.NET 10 Preview 1 includes new .NET SDK features & enhancements:
-
-- [Feature](#feature)
-
-.NET SDK updates in .NET 10:
-
-- [What's new in .NET 10](https://learn.microsoft.com/dotnet/core/whats-new/dotnet-10/overview) documentation
-
-
-## Feature
-
-This is something about the feature
+# .NET SDK in .NET 10 Preview 1 - Release Notes
+
+.NET 10 Preview 1 includes new .NET SDK features & enhancements:
+
+- [Pruning of Framework-provided Package References](#pruning-of-framework-provided-package-references)
+
+## Pruning of Framework-provided Package References
+
+Starting in .NET 10, the [NuGet Audit][nuget-audit] feature can now [prune framework-provided package references][prune-package-references-spec] that are not used by the project. This feature will be enabled by default for all .NET TargetFrameworks (e.g. `net8.0`, `net10.0`) and .NET Standard 2.0 and greater TargetFrameworks. This change will help reduce the number of packages that are restored and analyzed during the build process, which can lead to faster build times and reduced disk space usage. It also can lead to a reduction in false positives from NuGet Audit and other dependency-scanning mechanisms.
+
+When this feature is enabled, you should see a marked reduction in the contents of your applications' generated `.deps.json` files - any PackageReferences you may have had that are actually being supplied by the .NET Runtime you use will be automatically removed from the generated dependency file.
+
+While this feature is enabled by default for the TFMs listed above, you can disable it by setting the `RestoreEnablePackagePruning` property to `false` in your project file or Directory.Build.props file.
```

Comment:
I believe there is currently a warning included in 10 if you have a direct reference to a package that'll get pruned. Should that be included in these notes?

Reply:
I see a few warnings messages in the implementation: https://github.com/NuGet/NuGet.Client/pull/6142/files#diff-dfef6c5bcffe2d6c234f5d5797997b2f8d5277fda98340158a17241267975489R1130-R1141, are you thinking about
NU1510? I could try to make a scenario to show it here.

Reply:
I'll merge this now so there's some content, and work with our NuGet folks to get an example in later

Reply:
https://github.com/NuGet/docs.microsoft.com-nuget/commit/487e05961c0a8eccda5555a47eb2f38c365c3f5d. 

NU1509, NU1510 and NU1511 are all new warnings. 
We can say any of those may be raised as a consequence of this feature.